scorecard/serializers.py

```diff
@@ -19,6 +22,17 @@ def get_bbox(self, obj):
     class Meta:
         model = models.Geography
         exclude = ["id"]
+        # fields = ['bbox','is_disestablished','geo_level','geo_code','name','long_name','square_kms','parent_level','parent_code','province_name','province_code','category','miif_category',
+        #           'population','postal_address_1','postal_address_2','postal_address_3','street_address_1','street_address_2','street_address_3','street_address_4','phone_number','fax_number','url']
+        # extra_kwargs = {'is_disestablished': {'read_only':True}}
```

Comment:
I think this sort of comment can easily get out of date and be misleading. Is it still needed or can we delete it?